vmdb/app/controllers/host_controller.rb

```diff
@@ -837,19 +835,19 @@ def get_form_vars
   end
 
   def set_verify_status
-    if @edit[:new][:default_userid].blank? || @edit[:new][:ipaddress].blank?
+    if @edit[:new][:default_userid].blank?
```

Comment:
@h-kataria @jzigmund I don't know this code but should it be (ipaddress -> hostname).  

`if @edit[:new][:default_userid].blank? || @edit[:new][:hostname].blank?`

The idea is hostname is becoming the field used for communication so any place we lit up a button based on the ipaddress value, we need to verify if we need to put hostname there.